src/App.js

```diff
@@ -0,0 +1,23 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+
+import './App.css';
```

Comment:
- [ ] Please fix the import for the CSS it's an incorrect call for the App.css there is no such file, please take a good look to what you should call.

Reply:
I missed this. Thank you for the feedback